org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/navigation/XbaseHyperLinkHelper.java

```diff
@@ -108,6 +123,7 @@ public void setSourceViewer(ISourceViewer sourceViewer) {
 		List<IHyperlink> links = Lists.newArrayList();
 		IHyperlinkAcceptor acceptor = new XbaseHyperlinkAcceptor(links, createMultipleHyperlinks);
 		super.createHyperlinksByOffset(resource, offset, acceptor);
+		createHyperlinksInJavaDoc(resource, offset, acceptor);
```

Comment:
Wouldn't it be better to create it in `createHyperlinksByOffset(XtextResource, int, IHyperlinkAcceptor)`. If we're in a JavaDoc, `crossRefNode` is going to be null anyway so we'll call said method. It is also part of the public API meaning that if users use the method with the acceptor, they won't get the JavaDoc hyperlinks